about.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Rock&Roll</title>
+    <!-- stylesheet -->
+    <link rel="stylesheet" href="css/homepage.css" />
+    <link rel="stylesheet" href="css/aboutpage.css" />
+    <link rel="stylesheet" href="css/activity.css" />
+    <link rel="stylesheet" href="css/performer.css" />
+    <link rel="stylesheet" href="css/sponser.css" />
+    <link rel="stylesheet" href="css/footer.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <section id="header" class="header">
+      <!-----------------------------------First Header-------------------------------->
+      <header class="top-header">
```

Comment:
- [ ] Kindly make your header section on the mobile screen transparent or have the same pattern as the top section of your page, also, remove the box-shadow. You are expected to stick to design guidelines as much as you can.
![image](https://user-images.githubusercontent.com/94551484/227708755-53019180-4948-4b41-ad5c-c69452875ff2.png)

### Your design
![image](https://user-images.githubusercontent.com/94551484/227708723-f633f20a-2430-4387-a9ef-eee98c54ae71.png)

### Design Guidelines
![image](https://user-images.githubusercontent.com/94551484/227708739-a6aa88e7-5195-4546-9ba1-8153858b9af2.png)